src/utils/getLowResUrl.ts

```diff
@@ -1,12 +1,9 @@
 import { PixelRatio } from 'react-native';
 import { CardSize } from '../components/unique-token/CardSize';
+import { imageToPng } from '@rainbow-me/handlers/imgix';
 
-export const GOOGLE_USER_CONTENT_URL = 'https://lh3.googleusercontent.com/';
 const size = (Math.ceil(CardSize) * PixelRatio.get()) / 5;
 
 export const getLowResUrl = (url: string) => {
-  if (url?.startsWith?.(GOOGLE_USER_CONTENT_URL)) {
```

Comment:
before this only resized images stored via google, now we use imgix to resize all images

Reply:
casting to png for performance and so we can use reuse the same low res image for all loading states

Reply:
I agree with @osdnk  u can keep it as it was before and cast to png on the else